cmd/cork/create.go

```diff
@@ -262,9 +266,12 @@ func updateRepo() {
 }
 
 func runEnter(cmd *cobra.Command, args []string) {
-	enter := sdk.OldEnter
 	if experimental {
-		enter = sdk.Enter
+		plog.Warningf("--experimental is deprecated and does nothing. Fix your scripts.")
```

Comment:
The option is deprecated but the mode is now the default.  Also, the second sentence is redundant -- and there's no point in yelling at users who have been helpfully testing the new mode.

Reply:
I didn't mean to yell =(. I meant to just add some urgency since this doesn't exactly have a defined deprecation window. Maybe "--experimental is deprecated and does nothing. It will be removed next release" or "Update your scripts." (Since any scripts aren't _broken_ really, but do need to be updated)

Reply:
We can remove legacy mode without removing `--experimental` (at least for one value of `--experimental`, namely `true`) so I don't think there's a lot of urgency anyway.